<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapsVisualizer</class>
 <widget class="QMainWindow" name="MapsVisualizer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Maps Visualizer - MDT</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main.qrc">
    <normaloff>:/main/logo.gif</normaloff>:/main/logo.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>6</number>
           </property>
           <item>
            <widget class="QTabWidget" name="commandTabs">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="generalOptions">
              <property name="accessibleName">
               <string/>
              </property>
              <attribute name="title">
               <string>General</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_4">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <layout class="QGridLayout" name="generalTabPosition">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="mapOptions">
              <attribute name="title">
               <string>Maps</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QScrollArea" name="scrollArea_2">
                 <property name="horizontalScrollBarPolicy">
                  <enum>Qt::ScrollBarAlwaysOff</enum>
                 </property>
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="scrollAreaWidgetContents_2">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>977</width>
                    <height>657</height>
                   </rect>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_8" rowstretch="0,0,0,0,0,0,0,0,0">
                   <property name="leftMargin">
                    <number>6</number>
                   </property>
                   <property name="topMargin">
                    <number>6</number>
                   </property>
                   <property name="rightMargin">
                    <number>6</number>
                   </property>
                   <property name="bottomMargin">
                    <number>6</number>
                   </property>
                   <property name="spacing">
                    <number>6</number>
                   </property>
                   <item row="0" column="0" colspan="3">
                    <widget class="QComboBox" name="comboBox"/>
                   </item>
                   <item row="5" column="2">
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <item>
                      <widget class="QDoubleSpinBox" name="doubleSpinBox_3"/>
                     </item>
                     <item>
                      <widget class="QDoubleSpinBox" name="doubleSpinBox_4"/>
                     </item>
                    </layout>
                   </item>
                   <item row="8" column="0">
                    <spacer name="verticalSpacer">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>20</width>
                       <height>40</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item row="2" column="2">
                    <widget class="QLineEdit" name="lineEdit"/>
                   </item>
                   <item row="4" column="2">
                    <layout class="QHBoxLayout" name="horizontalLayout">
                     <item>
                      <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
                       <property name="specialValueText">
                        <string/>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
                     </item>
                    </layout>
                   </item>
                   <item row="4" column="0">
                    <widget class="QLabel" name="label_10">
                     <property name="text">
                      <string>Scale:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="2" column="0">
                    <widget class="QLabel" name="label_8">
                     <property name="text">
                      <string>Title:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="5" column="0">
                    <widget class="QLabel" name="label_11">
                     <property name="text">
                      <string>Clipping:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="1" column="0" colspan="3">
                    <widget class="Line" name="line">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                   <item row="6" column="0">
                    <widget class="QLabel" name="label_9">
                     <property name="text">
                      <string>Colormap:</string>
                     </property>
                    </widget>
                   </item>
                   <item row="6" column="2">
                    <widget class="QComboBox" name="comboBox_2"/>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="textInfoTab">
              <attribute name="title">
               <string>Textual</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_9">
               <item row="0" column="0">
                <widget class="TextConfigEditor" name="textConfigEdit"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="gridLayoutWidget">
          <layout class="QVBoxLayout" name="plotLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen_directory"/>
    <addaction name="separator"/>
    <addaction name="actionBrowse_to_current_folder"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_directory">
   <property name="text">
    <string>&amp;Open directory</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>&amp;Export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionExtra_plot_options">
   <property name="text">
    <string>&amp;Extra plot options</string>
   </property>
  </action>
  <action name="actionBrowse_to_current_folder">
   <property name="text">
    <string>&amp;Browse to current folder</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextConfigEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>..widgets</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>commandTabs</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>doubleSpinBox_2</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>doubleSpinBox_3</tabstop>
  <tabstop>doubleSpinBox_4</tabstop>
  <tabstop>comboBox_2</tabstop>
  <tabstop>textConfigEdit</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>scrollArea_2</tabstop>
 </tabstops>
 <resources>
  <include location="main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MapsVisualizer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
